metashare/api/views.py

```diff
@@ -42,9 +42,16 @@ class ProductViewSet(viewsets.ModelViewSet):
     serializer_class = ProductSerializer
     filter_backends = (DjangoFilterBackend,)
     filterset_class = ProductFilter
-    pagination_class = ProductPaginator
+    pagination_class = CustomPaginator
     model = Product
 
     def get_queryset(self):
         repositories = self.request.user.repositories.values_list("url", flat=True)
         return Product.objects.filter(repo_url__in=repositories)
+
+
+class ProjectViewSet(viewsets.ModelViewSet):
+    permission_classes = (IsAuthenticated,)
+    serializer_class = ProjectSerializer
+    pagination_class = CustomPaginator
+    queryset = Project.objects.all()
```

Comment:
@wlonk Are these ordered by most-recently-created-first? I think that's the desired functionality, at least for now.

Reply:
Implicitly, yes. They should get an explicit ordering, though.